3-extra/1-radio-stations.js

```diff
@@ -14,6 +14,15 @@
  */
 
 // `getAllFrequencies` goes here
+function getAllFrequencies () {
```

Comment:
This works 👍 
But can you think of another way to do this? What if we had thousands of frequencies - we might not want to list them all out one by one.